shared/app/nav.native.js

```diff
@@ -116,41 +110,32 @@ const barStyle = (showStatusBarDarkContent, underStatusBar) => {
   if (showStatusBarDarkContent) {
```

Comment:
Maybe we should change the semantics of this to `showStatusBarLightContent`. there's no way to set the content to light on iOS. we'd want to do that on the attachment fullscreen page.
<img src="https://user-images.githubusercontent.com/11968340/48072842-69791400-e1ab-11e8-9b94-dbc1b5df7fc3.jpg" width="350" />